blackbox/docs/release_notes/1.1.6.txt

```diff
@@ -0,0 +1,64 @@
+.. _version_1.1.6:
+
+=============
+Version 1.1.6
+=============
+
+Released on 2017/06/23.
+
+.. NOTE::
+
+   If you are upgrading a cluster, you must be running CrateDB 0.57.0 or higher
```

Comment:
On master + 2.x (I think) the wording of this section is different, and the section refs have changed. It needs to be
```
.. NOTE::

   If you are upgrading a cluster, you must be running CrateDB 0.57.0 or higher
   before you upgrade to 1.1.6.

   If you want to perform a :ref:`rolling_upgrade`, your current CrateDB
   version number must be :ref:`version_1.1.1` or higher. If you want to
   upgrade from a version prior to this, the upgrade will introduce all of the
   breaking changes listed for :ref:`version_1.1.0`, and will require a
   :ref:`full_restart_upgrade`.
```

The previous PR for the docs on the 1.1 branch are correct - this is just for master